src/sql/src/plan/statement/dml.rs

```diff
@@ -956,12 +956,18 @@ fn plan_copy_to(
 
     let to = plan_expr(ecx, &to_expr)?.type_as(ecx, &ScalarType::String)?;
 
+    if options.max_file_size.as_bytes() < ByteSize::mb(16).as_bytes() {
+        sql_bail!("MAX FILE SIZE cannot be less than 16MB");
```

Comment:
Do we actually care? looks like an artificial limit

Reply:
If it's too small, say something like 1B, we can end up writing too many files to s3. Would be bad for performance.